error: 'NLGA' undeclared here (not in a function)

I'm installed libunwind on centos 7.4 86_64x ,but it's  error:

In file included from x86_64/Linit.c:4:0:
x86_64/Ginit.c:216:28: error: 'NLGA' undeclared here (not in a function)
 static AO_T last_good_addr[NLGA];
                            ^
x86_64/Ginit.c:216:13: warning: 'last_good_addr' defined but not used [-Wunused-variable]
 static AO_T last_good_addr[NLGA];
             ^
make[2]: *** [x86_64/Linit.lo] Error 1
make[2]: Leaving directory `/usr/local/src/libunwind/src'
make[1]: *** [all] Error 2
make[1]: Leaving directory `/usr/local/src/libunwind/src'
make: *** [all-recursive] Error 1

--- a/src/x86_64/Ginit.c
+++ b/src/x86_64/Ginit.c
@@ -174,8 +174,8 @@ tdep_init_mem_validate (void)
 }
 
 /* Cache of already validated addresses */
-#if defined(HAVE___THREAD) && HAVE___THREAD
 #define NLGA 4
+#if defined(HAVE___THREAD) && HAVE___THREAD
 // thread-local variant
 static __thread unw_word_t last_good_addr[NLGA];
 static __thread int lga_victim;
@@ -394,11 +394,6 @@ x86_64_local_addr_space_init (void)
   local_addr_space.acc.resume = x86_64_local_resume;
   local_addr_space.acc.get_proc_name = get_static_proc_name;
   unw_flush_cache (&local_addr_space, 0, 0);
-
-#if NLGA > 0
-  memset (last_good_addr, 0, sizeof (unw_word_t) * NLGA);
-  lga_victim = 0;
-#endif
 }
 
 #endif /* !UNW_REMOTE_ONLY */

